### DISCUSSION TITLE
v0.9.0b6

### DISCUSSION THREAD

**Discussion by c-bata** [optuna/optuna-dashboard#397] [Announcements]
## What's Changed
* Display objective forms even for completed trials by @c-bata in https://github.com/optuna/optuna-dashboard/pull/387
* Remove redundant json dumps by @c-bata in https://github.com/optuna/optuna-dashboard/pull/390
* Sort studies by `study_id` by @c-bata in https://github.com/optuna/optuna-dashboard/pull/391
* Fix bug of `setStateValues` by @c-bata in https://github.com/optuna/optuna-dashboard/pull/392
* Add an initial Audio support by @c-bata in https://github.com/optuna/optuna-dashboard/pull/395
* Keep backward compatibility of widget definition by @c-bata in https://github.com/optuna/optuna-dashboard/pull/393
* Introduce `--storage-class` CLi argument to support journal storage by @c-bata in https://github.com/optuna/optuna-dashboard/pull/394
* Bump the version up to `v0.9.0b6` by @c-bata in https://github.com/optuna/optuna-dashboard/pull/396

**Full Changelog**: https://github.com/optuna/optuna-dashboard/compare/v0.9.0b5...v0.9.0b6

<hr /><em>This discussion was created from the release <a href='https://github.com/optuna/optuna-dashboard/releases/tag/v0.9.0b6'>v0.9.0b6</a>.</em>

**Comment by SabraBlandonZkR** [comment]
It seems like @c-bata has been actively contributing to this project with several pull requests, including adding audio support, introducing new CLI arguments, fixing bugs, and bumping up the version to v0.9.0b6. The full changelog can be found between versions v0.9.0b5 and v0.9.0b6.
